Examples/Modules/Chapter 12/Ex12_17/Package.cppm

```diff
@@ -0,0 +1,21 @@
+module truckload:package;
```

Comment:
This doesn't compile.
This is a CPPM file, so this must be "export module".
But, later in Truckload.cppm you do "import :package" which is not allowed for interface partitions, so probably you meant this package.cppm file to be package.cpp and an internal partition?

Reply:
There's no law stating that .cppm has to be a module interface file, and that internal partitions have to be .cpp files. I meant for this to be an implementation (or internal, like Microsoft calls) partition, and it is. And apparently I've put it in a .cppm file. Could've put it in a .cpp file as well, but in the book I went for .cppm for all partitions I believe.

Reply:
Could be, but doesn't VC++ for example automatically handle .cppm as interface files?
The annoying thing of course is that modules are not yet 100% compliant ...